controllers/syncer/csi_syncer.go

```diff
@@ -631,81 +575,36 @@ func (s *csiControllerSyncer) ensureVolumes() []corev1.Volume {
 	}
 }
 
-func (s *csiControllerSyncer) getSidecarByName(name string) *csiv1.CSISidecar {
-
-	logger := csiLog.WithName("getSidecarByName")
-	logger.Info("in getSidecarByName")
-
-	return getSidecarByName(s.driver, name)
-}
-
-func (s *csiControllerSyncer) getSidecarImageByName(name string) string {
-
-	logger := csiLog.WithName("getSidecarImageByName")
-	logger.Info("in getSidecarImageByName", "name", name)
-
-	sidecar := s.getSidecarByName(name)
-	if sidecar != nil {
-		return fmt.Sprintf("%s:%s", sidecar.Repository, sidecar.Tag)
-	}
-	logger.Info("didn't find sidecar image")
-	return s.driver.GetDefaultSidecarImageByName(name)
-}
-
-func (s *csiControllerSyncer) getCSIAttacherImage() string {
-	logger := csiLog.WithName("getCSIAttacherImage")
-	logger.Info("in getCSIAttacherImage")
-
-	return "us.gcr.io/k8s-artifacts-prod/sig-storage/csi-attacher:v3.3.0"
-	//return s.getSidecarImageByName(config.CSIAttacher)
-}
-
-func (s *csiControllerSyncer) getCSIProvisionerImage() string {
-	return "us.gcr.io/k8s-artifacts-prod/sig-storage/csi-provisioner:v3.0.0"
-	//return s.getSidecarImageByName(config.CSIProvisioner)
-}
-
-func (s *csiControllerSyncer) getLivenessProbeImage() string {
-	return "us.gcr.io/k8s-artifacts-prod/sig-storage/livenessprobe:v2.3.0"
-	//return s.getSidecarImageByName(config.LivenessProbe)
-}
-
-func (s *csiControllerSyncer) getCSISnapshotterImage() string {
-	return "us.gcr.io/k8s-artifacts-prod/sig-storage/csi-snapshotter:v4.1.1"
-	//return s.getSidecarImageByName(config.CSISnapshotter)
-}
-
-func (s *csiControllerSyncer) getCSIResizerImage() string {
-	return "us.gcr.io/k8s-artifacts-prod/sig-storage/csi-resizer:v1.3.0"
-	//return s.getSidecarImageByName(config.CSIResizer)
-}
+func (s *csiControllerSyncer) getSidecarImage(name string) string {
+	logger := csiLog.WithName("getSidecarImage")
```

Comment:
let's add a doc string, stating the purpose of the method and return type.

Reply:
Same as above, Thanks!